tools/authors/manage.php

```diff
@@ -468,7 +478,11 @@ function evaluateForm(form) {
 
 while ($i++ < $count && $author = @mysqli_fetch_array($result)) {
     $id = $author['author_id'];
-    $bioresult = mysqli_query(DPDatabase::get_connection(), "SELECT bio_id FROM biographies WHERE author_id = $id ORDER BY bio_id;");
+    $bioresult = DPDatabase::query(
```

Comment:
Break this out into an `$sql=`. $id doesn't need to be escape()d. query() doesn't need a 'false'.